test/instrumentation/modules/elasticsearch.js

```diff
@@ -59,6 +59,92 @@ test('client.search with callback', function userLandCode (t) {
   })
 })
 
+test('client.searchTemplate with callback', function userLandCode (t) {
+  var body = {
+    source: {
+      query: {
+        query_string: {
+          query: '{{q}}'
+        }
+      }
+    },
+    params: {
+      q: 'pants'
+    }
+  }
+
+  resetAgent(done(t, 'POST', '/_search/template', JSON.stringify(body)))
+
+  agent.startTransaction('foo3')
```

Comment:
I honestly can't remember why I added the number suffix to the transaction names, but in all the previous tests I incremented it by one in each test. So I think either we should remove the suffix completely or we should continue the pattern in the new tests 🤷‍♂️

Reply:
Ah, right. 😅

Reply:
fixed